src/app/components/profile/profile.component.spec.ts

```diff
@@ -10,19 +10,62 @@ import { NavbarComponent } from '../navbar/navbar.component';
 import { PreferenceComponent } from '../preference/preference.component';
 import { HttpClientModule } from '@angular/common/http';
 import { AppRoutingModule } from 'src/app/app-routing.module';
-import { FormsModule } from '@angular/forms';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { APP_BASE_HREF } from '@angular/common';
 import { CarRegisterComponent } from '../car-register/car-register.component';
 import { userInfo } from 'os';
+import { DriverComponent } from '../driver/driver.component';
+import { AdminLoginComponent } from '../admin-login/admin-login.component';
+import { DriverInfoComponent } from '../driver-info/driver-info.component';
+import { LandingComponent } from '../landing/landing.component';
+import { SignupModalComponent } from '../sign-up-modal/sign-up-modal.component';
+import { LandingPageComponent } from '../landing-page/landing-page.component';
+import { ProfileContactComponent } from '../profile-contact/profile-contact.component';
+import { ProfileCarComponent } from '../profile-car/profile-car.component';
+import { ProfileLocationComponent } from '../profile-location/profile-location.component';
+import { ProfileMembershipComponent } from '../profile-membership/profile-membership.component';
+import { DriverContactModalComponent } from '../driver-contact-modal/driver-contact-modal.component';
+import { DriverListComponent } from '../driver-list/driver-list.component';
+import { HomePageComponent } from '../home-page/home-page.component';
+import { BsNavbarComponent } from 'src/app/bs-navbar/bs-navbar.component';
+import { ScheduleComponent } from '../schedule/schedule.component';
+import { ViewMyRidesComponent } from '../view-my-rides/view-my-rides.component';
 
 describe('ProfileComponent', () => {
   let component: ProfileComponent;
   let fixture: ComponentFixture<ProfileComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ ProfileComponent, AdminComponent, UserRegisterComponent, LoginComponent, MyCarComponent, NavbarComponent, PreferenceComponent, CarRegisterComponent ],
-      imports: [HttpClientModule, AppRoutingModule, FormsModule],
+      declarations:[
+        RegisterComponent,
+        DriverComponent,
+        AdminComponent,
+        LoginComponent,
+        CarRegisterComponent,
+        LoginComponent,
+        NavbarComponent,
+        MyCarComponent,
+        ProfileComponent,
+        PreferenceComponent,
+        AdminLoginComponent,
+        DriverInfoComponent,
+        LandingComponent,
+        SignupModalComponent,
+        LandingPageComponent,
+        ProfileContactComponent,
+        ProfileCarComponent,
+        ProfileLocationComponent,
+        ProfileMembershipComponent,
+        DriverContactModalComponent,
+        DriverListComponent,
+        UserRegisterComponent,
+        HomePageComponent,
+        BsNavbarComponent,
+        ScheduleComponent,
+        ViewMyRidesComponent,
```

Comment:
Indeed these are unfortunate. Is it possible to have several listed in the same line to conserve some vertical space? What are some alternatives that have been tried so far?